src/main/resources/static/app.js

```diff
@@ -0,0 +1,151 @@
+let stompClient = null;
+
+function setConnected() {
+    $("#buttonConnect").addClass("d-none");
+    $("#buttonDisconnect").removeClass("d-none");
+    $("#buttonConnecting").addClass("d-none");
+
+    $("#alertBox").removeClass();
+    $("#alertBox").addClass("alert alert-success");
+    $("#alertBox").text("Connected :)");
+}
+
+function setDisconnected() {
+    $("#buttonConnect").removeClass("d-none");
+    $("#buttonDisconnect").addClass("d-none");
+    $("#buttonConnecting").addClass("d-none");
+
+    $("#alertBox").removeClass();
+    $("#alertBox").addClass("alert alert-warning");
+    $("#alertBox").text("Server not connected!");
+}
+
+function setConnectionPending() {
+    $("#buttonConnect").addClass("d-none");
+    $("#buttonDisconnect").addClass("d-none");
+    $("#buttonConnecting").removeClass("d-none");
+}
+
+function connect() {
+    setConnectionPending();
+
+    let socket = new SockJS('/gs-guide-websocket');
+    stompClient = Stomp.over(socket);
+    stompClient.connect(
+        {}, // headers
+        function (frame) { // connectCallback
+            setConnected();
+            console.log('Connected: ' + frame);
+
+            stompClient.subscribe('/generation', function (generationData) {
+                drawCellsFromData(JSON.parse(generationData.body).generationData);
```

Comment:
Unresolved variable `generationData` (the one at the end)

Reply:
I don´t know how to handle it easy ... we are getting a JSON object with this property ... but JavaScript / IDE does not know that this property is available.

Reply:
Ok, then forget about it - we don't need to handle it.